modules/test/playwright/pages/product-navigation-applications-menu/ApplicationsMenuPage.ts

```diff
@@ -182,11 +182,13 @@ export class ApplicationsMenuPage {
 		});
 	}
 
-	async goto() {
+	async goto(checkTabVisibility = true) {
```

Comment:
same here, I don't know if you have to specify the parameter as optional (with the ?)